app/helpers/permits_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PermitsHelper
```

Comment:
Will we need this file for something?

Reply:
These are typically used for helper functions for the views if you are making erb templates. It was generated by default with scaffold.